Source/WebKit/WebProcess/FullScreen/WebFullScreenManager.cpp

```diff
@@ -387,7 +387,7 @@ void WebFullScreenManager::handleEvent(WebCore::ScriptExecutionContext& context,
     }
 
     if (targetElement == m_mainVideoElement.get()) {
-        if (m_mainVideoElement && m_mainVideoElement->paused())
+        if (m_mainVideoElement->paused() && !m_mainVideoElement->seeking())
```

Comment:
NIT: would it be better to use `targetElement` instead to avoid doing a double dereference (via `WeakPtr`)?

Reply:
Makes sense! Will change to `targetElement` here.